### ISSUE TITLE
Chome 102: has been blocked by CORS policy

### ISSUE THREAD

**Issue by W84Soft** [neutralinojs/neutralinojs#925]
Hi, I have a problem with new version of webview2 (Chrome 102), I can't even load any script
<img width="469" alt="Zrzut ekranu 2022-06-09 233104" src="https://user-images.githubusercontent.com/53473441/172948795-57ed7273-5794-40f4-abbd-ec74f0ebbb2a.png">

Can I set flag --disable-web-security to modes.window ?

**Comment by shalithasuranga** [comment]
Closing as duplicate of #586. Thanks for reporting this issue :tada:
